docs/source/faq.rst

```diff
@@ -164,7 +163,7 @@ monkey patching of methods.
 How is mypy different from Cython?
 **********************************
 
-`Cython :doc:<cython:index>` is a variant of Python that supports
+:doc:`Cython <cython:index>` is a variant of Python that supports
```

Comment:
Fixing my own errors here. No idea why Sphinx didn't catch this on build.

Reply:
Didn't you have to run it with special flags to check errors?

Reply:
I did (using `sphinx-build -WEan`), however Sphinx ignores markup errors and just renders the text as-is. Neither does `doc8` nor `restructuredtext-lint`, sadly.